src/Cake.Core/Polyfill/EnvironmentHelper.cs

```diff
@@ -119,7 +119,7 @@ public static Runtime GetRuntime()
         public static FrameworkName GetBuiltFramework()
         {
 #if NETCORE
-            return new FrameworkName(".NETStandard,Version=v2.0");
+            return new FrameworkName(".NETCoreApp,Version=v2.1");
```

Comment:
Is Cake really `2.1`? Shouldn't this be `v2.0`?

Reply:
Cake is 2.0 and Cake.Tool is 2.1.

Reply:
Yeah, but shouldn't this parameter reflect the framework we actually built Cake on?

Reply:
Could we query runtime? As you may run on newer?

Reply:
Maybe this could help? https://weblog.west-wind.com/posts/2018/Apr/12/Getting-the-NET-Core-Runtime-Version-in-a-Running-Application I.e. doable at runtime, via reflection.

Reply:
No., the runtime version is not of interest here. It's the target framework of Cake. And like the property says, this should reflect what it's built target framework is.

Reply:
AFAIK, that is the built framework.
![image](https://user-images.githubusercontent.com/7863439/45874648-18999180-bd9e-11e8-9759-24f3977207ee.png)

Reply:
Yes. `.NetCoreApp, Version=v2.0`

Reply:
And for `Cake.Tool` it shows `.NetCoreApp, Version=v2.1`. So, we can use this approach, right?

Reply:
Is Cake.dll in Cake.Tool compiled for 2.1?

Reply:
I mean does the Cake.dll have the 2.1 target framework or is the package just 2.1?

Reply:
Yes, it’s Cake.exe/Cake.dll I’m referring to.